libsplinter/src/oauth/rest_api/actix/callback.rs

```diff
@@ -83,15 +90,57 @@ pub fn make_callback_route(
                                     {
                                         Ok(session) => {
                                             match oauth_user_session_store.add_session(session) {
-                                                Ok(_) => HttpResponse::Found()
+                                                Ok(_) => {
```

Comment:
We probably shouldn't nest this in the `add_session` match. `cargo fmt` can't handle this because the nesting is too deep for it to properly format these lines.

Reply:
Fixed